content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

```diff
@@ -181,6 +181,19 @@ Whether log messages are printed for the meta service.
 
 Environment variable: `INFLUXDB_META_LOGGING_ENABLED`
 
+#### `password-hash = bcrypt`
+
+Configures password hashing algorithm.
+Supported options are: `bcrypt` (the default), `pbkdf2-sha256`, and `pbkdf2-sha512`
+This setting must have the same value as the meta node option [`meta.password-hash`](/enterprise_influxdb/v1.9/administration/config-meta-nodes/#password-hash--bcrypt).
+When not set, InfluxDB Enterprise uses `bcrypt`.
+`password-hash` is not set by default.
```

Comment:
```suggestion
```

Reply:
think line 190 is inaccurate given default is bcrypt?

Reply:
This is accurate as far as I know., but might be confusing.

In the default configuration file, the `password-hash` setting is commented out ("not set"). When commented out, `bcrypt` is the default. To change the algorithm, uncomment and change the value.

Maybe I should say that :)

Reply:
I thought about it more. Your suggestion is better.